packages/babel-parser/typings/babel-parser.source.d.ts

```diff
@@ -1,177 +1,24 @@
-// Type definitions for @babel/parser
```

Comment:
I left this file here to avoid accidental breaking changes, but now it's just re-exporting form the source.